### PR TITLE
added collator instructions

### PR DESCRIPTION
**Pull request by artkaseman**
First draft of collator instructions.  I just copied and pasted the full node instructions and added what's needed for collation.

### CODE REVIEW

File: node-operators/networks/collator.md [hunk 1]
```diff
@@ -0,0 +1,332 @@
+---
+title: Collator
+description: How to run a full Parachain node for the Moonbeam Network to have your own RPC Endpoint
```
Comment (JoshOrndorff) [inline node-operators/networks/collator.md:3]:
```suggestion
description: How to run a Collator node for the Moonbeam Network to author blocks
```

File: node-operators/networks/collator.md [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+---
+
+# Run a Collator on Moonbeam
+
+![Full Node Moonbeam Banner](/images/fullnode/fullnode-banner.png)
+
+## Introduction
+
+With the release of Moonbase Alpha v6, you can spin up a collator that connects to the Moonbase Alpha TestNet, syncs with a bootnode, and provides local access your own RPC endpoints.
```
Comment (JoshOrndorff) [inline node-operators/networks/collator.md:12]:
```suggestion
With the release of Moonbase Alpha v6, you can spin up a collator that connects to the Moonbase Alpha TestNet, syncs with a bootnode, and provides local access your own RPC endpoints, and authors blocks in the parachain. 
```

File: node-operators/networks/collator.md [hunk 3]
```diff
@@ -0,0 +1,332 @@
...
+
+|  Description |   |                Port                  |
+|:------------:|:-:|:------------------------------------:|
+|    **P2P**   |   | {{ networks.relay_chain.p2p }} (TCP) |
+|    **RPC**   |   | {{ networks.relay_chain.rpc }}       |
+|    **WS**    |   | {{ networks.relay_chain.ws }}        |
+|**Prometheus**|   | {{ networks.relay_chain.prometheus }}|
+ 
+
+## Installation Instructions - Generate an Account
```
Comment (JoshOrndorff) [inline node-operators/networks/collator.md:71]:
```suggestion
## Prep Work - Generate an Account
```

File: node-operators/networks/collator.md [hunk 4]
```diff
@@ -0,0 +1,332 @@
...
+purestake/moonbase-parachain-testnet:{{ networks.moonbase.parachain_docker_tag }} \
+/moonbase-alphanet/moonbase-alphanet \
+    --base-path=/data \
+    --chain alphanet \
+    --name="YOUR-NODE-NAME" \
+    --collator \ 
+    --author-id PUBLIC_KEY \
+    --execution wasm \
+    --wasm-execution compiled \
+    --state-cache-size {{networks.moonbase.state-cache-size }} \
```
Comment (JoshOrndorff) [inline node-operators/networks/collator.md:106]: no longer necessary

File: node-operators/networks/collator.md [hunk 5]
```diff
@@ -0,0 +1,332 @@
...
+     --pruning=archive \
+     --unsafe-rpc-external \
+     --unsafe-ws-external \
+     --rpc-methods=Safe \
+     --rpc-cors all \
+     --log rpc=info \
+     --base-path {{ networks.moonbase.node_directory }} \
+     --chain alphanet \
+     --name "YOUR-NODE-NAME" \
+     --state-cache-size {{networks.moonbase.state-cache-size }} \
```
Comment (JoshOrndorff) [inline node-operators/networks/collator.md:237]: no longer necessary

**Review by JoshOrndorff — COMMENTED** [review]
Just a few things I noticed. Looking good.

File: node-operators/networks/collator.md (hunk 4 above)
Comment (artkaseman) [reply node-operators/networks/collator.md]: updated in latest

File: node-operators/networks/collator.md (hunk 5 above)
Comment (artkaseman) [reply node-operators/networks/collator.md]: updated in latest

File: node-operators/networks/collator.md (hunk 1 above)
Comment (artkaseman) [reply node-operators/networks/collator.md:3]: updated

File: node-operators/networks/collator.md [hunk 6]
```diff
@@ -0,0 +1,328 @@
...
+
+```
+journalctl -f -u moonbeam.service
+```
+
+![Service Logs](/images/fullnode/fullnode-binary3.png)
+
+## Telemetry
+
+_Comming soon_
```
Comment (4meta5) [inline node-operators/networks/collator.md:273]:
```suggestion
_Coming soon_
```

**Pull request merged**